srcpkgs/satty/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'satty'
+pkgname=satty
+version=0.12.0
+revision=1
+build_style=cargo
+hostmakedepends="glib-devel pkg-config libadwaita-devel gtk4-devel gdk-pixbuf-devel libepoxy-devel fontconfig"
+makedepends="glib libadwaita gtk4 gdk-pixbuf fontconfig libepoxy"
+depends=" glib libadwaita gtk4 gdk-pixbuf fontconfig libepoxy"
```

Comment:
You don't need this line, xbps automatically detects these runtime dependencies

Reply:
Oh ok, did not know that

Reply:
Oh no no, sorry, I don't know why my comment referenced lines 5-8: I wrote the comment only referring to line 8. You do need lines 5-7, xbps doesn't pull in make depends but does pull in runtime depends. Apologies for the confusion.

Reply:
No worries. I am fixing it right up!